packages/apex-node/typings/modules/faye/index.d.ts

```diff
@@ -4,19 +4,11 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-declare module 'faye' {
-  export class Client {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    constructor(topic: string, options?: Record<string, any>);
-    setHeader(name: string, value: string): void;
-    on(event: string, callback: () => void): void;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    addExtension(extension: Record<string, any>): void;
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    subscribe(channel: string, callback: (message: any) => void): Subscription;
-    handshake(callback: () => void): void;
-    disconnect(): void;
-  }
 
-  export class Subscription {}
+import { Client } from 'faye';
+
+export class ApexFayeClient extends Client {
+  on(event: string, callback: () => void): void;
+  handshake(callback: () => void): void;
+  disconnect(): void;
```

Comment:
Why aren't these included in the Client form the 'module' found in JSForce? no idea.